app/assets/images/payment_icons/abitab.svg

```diff
@@ -0,0 +1,6 @@
+<svg
```

Comment:
Hi, this comment applies on all other icons:

Please follow the [contributions](https://github.com/activemerchant/payment_icons/blob/master/CONTRIBUTING.md) guideline:
- The background is missing
- The border of the icon is missing
- Please run the SVG code through svgo

You can follow this [example icon](https://github.com/activemerchant/payment_icons/blob/master/app/assets/images/payment_icons/visa.svg) as a reference.